src/pages/blog.js

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
```

Comment:
I think this page is not necessary, we only need to show all articles links in the main page